hr_holidays_accrual_advanced/README.rst

```diff
@@ -34,6 +34,7 @@ introducing following extra features:
  * Various accrual methods
```

Comment:
* Various accrual methods:
    * Start/End of Period
    * Prorate
    * Start/End of Month